packages/cli/src/util/lockfile.ts

```diff
@@ -1,37 +1,45 @@
-export type Lockfile = {
-  lockSync(path: string): void;
-  unlockSync(path: string): void;
-};
-
-const lockFile: Lockfile = (await import("lockfile")) as Lockfile;
-
-function getLockFilepath(filepath: string): string {
-  return `${filepath}.lock`;
-}
-
-/**
- * When lockfile is imported, it registers listeners to process
- * Since it's only used by the validator client, require lazily to not pollute
- * beacon_node client context
```

Comment:
This statement was not correct since the migration to ESM in https://github.com/ChainSafe/lodestar/pull/3978. It was also already noted in https://github.com/ChainSafe/lodestar/pull/3978#discussion_r877769141 but having async file locking causes a bunch of upstream issues.